### PR TITLE
Return t after calling service in (call-empty-service service)

### PR DESCRIPTION
**Pull request by YutoUchimi**
Related to https://github.com/jsk-ros-pkg/jsk_pr2eus/pull/396

Also, removed unmatched parenthesis in roseus-utils.l (similar fix was #607)

### CODE REVIEW

**Comment by Affonso-Gui** [comment]
> How about we include the `(if (ros::wait-for-service name 0) ...)` suggested in [jsk-ros-pkg/jsk_pr2eus#396](https://github.com/jsk-ros-pkg/jsk_pr2eus/pull/396) here and then just make a simple call to `call-empty-service` on the costmap PR?

Added PR in https://github.com/YutoUchimi/jsk_roseus/pull/1

**Comment by YutoUchimi** [comment]
Waiting for #622 to pass travis test.

**Comment by Affonso-Gui** [comment]
@YutoUchimi @k-okada Seems to be passing now!

File: roseus/euslisp/roseus-utils.l [hunk 1]
```diff
@@ -1039,10 +1039,11 @@
 ;; misc function
 ;;
 (ros::roseus-add-srvs "std_srvs")
-(defun call-empty-service (srvname &key (wait nil))
-  "Call empty service"
-  (if wait (ros::wait-for-service srvname))
-  (ros::service-call srvname (instance std_srvs::EmptyRequest :init)))
+(defun call-empty-service (srvname &key (wait nil) (timeout -1)) ;; If timeout is -1 (default), waits until the node is shutdown. If timeout is 0, returns immediately.
```
Comment (k-okada) [inline roseus/euslisp/roseus-utils.l:1042]: @Affonso-Gui 
`, waits until the node is shutdown` is this correct?
`, waits until the service becomes available` ?

Comment (Affonso-Gui) [reply roseus/euslisp/roseus-utils.l:1042]: Waits until either the service becomes available or the node is shutdown.

This part is directly from the rosscpp documentation, so I don't think this is a problem.
```
If timeout is -1 (default), waits until the node is shutdown
```

http://docs.ros.org/en/noetic/api/roscpp/html/namespaceros_1_1service.html#aabe996581255345b3383e66eaaedef5a

In a separate note, timeout=-1 will NOT respond to C-c, which can lead to disappointing results in the interpreter.

**Pull request merged**